### ISSUE TITLE
Cannot select value from a custom Datepicker column

### ISSUE THREAD

**Issue by javier-godoy** [vaadin/vaadin-grid-pro#149]
Vaadin version: 14.4.2 

Observed behavior:
* The value is updated when typing the value
* The value is updated when opening the popup and pressing key up / key down, then selecting a value using the mouse
* The value is NOT updated when selecting a value from the popup, without using the keyboard

(this is unrelated to #95 since it describes some styling issues)

### Example
```
GridPro<Item> gridpro = new GridPro<>();
DatePicker datePicker = new DatePicker();
gridpro.addEditColumn(item->item.getValue(), new LocalDateRenderer<>(item->item.getValue(), "dd/MM/yyyy")).custom(datePicker,(item,date) -> {
  item.setValue(date);
  System.out.println(date);
});
gridpro.setItems(new Item());
```

**Comment by tomivirkki** [comment]
Hi,
I tried with the snippet you provided but was able to change the DatePicker (and the Item) value by using mouse only. Can you attach a repro project as a zip file?

**Comment by javier-godoy** [comment]
@tomivirkki Please find attached. I reproduced it in both Chrome and Firefox.

[skeleton-starter-flow-14.zip](https://github.com/vaadin/vaadin-grid-pro/files/5895929/skeleton-starter-flow-14.zip)
![grid-date](https://user-images.githubusercontent.com/11554739/106319604-72ed6b80-6250-11eb-879a-3dbcd2918697.gif)

**Comment by tomivirkki** [comment]
Thanks for the attachment,
seems that the microtask timing [used for closing the editor after a `focusout` event](https://github.com/vaadin/vaadin-grid-pro/blob/928295525a375fd912ef69fdef17980cf79fefe1/src/vaadin-grid-pro-inline-editing-mixin.js#L262) is too eager.

**Comment by DiegoCardoso** [comment]
Taking a look into [this comment](https://github.com/vaadin/vaadin-grid-pro-flow/issues/85#issuecomment-600020607) from the issue fixed by #111, @Haprog tried with different approaches and `animationFrame` worked for that issue: 

> * `Polymer.Async.microTask`
>   * Save button without workaround **seems to work always** and all web component tests pass
> * `Polymer.Async.animationFrame`
>   * Save button without workaround **seems to work always** and all web component tests pass

Trying it locally seemed to work for `DatePicker` as well.
